### DISCUSSION TITLE
how to dowload 5.3.0

### DISCUSSION THREAD

**Discussion by Muthuvel2019** [mitmproxy/mitmproxy#5875] [Q&A] [answered]
where to download binary or source of mitmproxy 5.3.0.

Thank you

**Comment by mhils** [comment, marked as answer]
You can find older binaries at https://mitmproxy.org/downloads/, but we provide absolutely _zero_ support for that use case.

Please see: https://github.com/mitmproxy/mitmproxy/blob/main/SECURITY.md#supported-versions
